src/Razor/src/Microsoft.VisualStudio.RazorExtension.Dependencies/Microsoft.VisualStudio.RazorExtension.Dependencies.csproj

```diff
@@ -85,6 +89,7 @@
     <VSIXSourceItem Include="$(OutputPath)Microsoft.VisualStudio.LanguageServices.dll" />
     <VSIXSourceItem Include="$(OutputPath)Microsoft.VisualStudio.LanguageServices.CSharp.dll" />
     <VSIXSourceItem Include="$(OutputPath)Microsoft.VisualStudio.LanguageServices.Implementation.dll" />
+    <VSIXSourceItem Include="$(OutputPath)Microsoft.VisualStudio.Threading.dll" />
```

Comment:
Why is this needed? Is the version shipping in Visual Studio not enough?

Reply:
This project deploys roslyn and dependencies to the exp hive that normally come from VS. Most of the time its not needed, but when we have bumps Roslyn past what is in the VS preview version that our PR integration test leg runs on (currently 17.7 P1) then sometimes we need to use this to get things working. In this specific case, this project already had a binding redirect for MS.VS.Threading to 17.7, but nothing deployed it (and the version in the VS preview version was not sufficient). This got me past the integration tests hanging locally, by fixing the crash at startup of the experimental hive, due to not finding the 17.7 version of this DLL.